features/git-ship/current_branch/merge_main_branch_conflict.feature

```diff
@@ -12,7 +12,14 @@ Feature: git ship: resolving conflicts between feature and main branch
       | main    | local    | conflicting main commit    | conflicting_file | main content    |
       | feature | local    | conflicting feature commit | conflicting_file | feature content |
     And I am on the "feature" branch
-    And I run `git ship -m "feature done"` while allowing errors
+    And I run `git ship -m "feature done"`
+    Then it errors and the output ends with
```

Comment:
Yeah, I think a limitation of an early implementation required this but I believe I can now just check it there